src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -3,18 +3,30 @@ import { Button } from "./Button";
 import { Description } from "./Description";
 import { Image } from "./Image";
 import { Name } from "./Name";
+import { Date } from "./Date";
 
 interface IProps {
+  date?: string;
   description: string;
   name: string;
   icon: string;
   alt: string;
+  button: string;
 }
-export const Thumbnail: FC<IProps> = ({ description, name, icon, alt }) => (
-  <div className="p-8 w-full shadow" style={{ height: "304px" }}>
-    <Image icon={icon} alt={alt} classname="mb-2" />
-    <Name>{name}</Name>
-    <Description className="mb-16">{description}</Description>
-    <Button>Zistiť viac</Button>
+
+export const Thumbnail: FC<IProps> = ({
+  description,
+  name,
+  icon,
+  alt,
+  date,
+  button
+}) => (
+  <div className="p-8 shadow h-full flex-col flex justify-between">
```

Comment:
this whole div could be `font-lato`, you're adding it unnecessarily in every component